concepts/05 UI Components/DataGrid/99 How To/Bind a Lookup Column to a Custom Data Source.md

```diff
@@ -201,6 +300,147 @@ The following alternative **CustomStore** configuration delegates data processin
         <!-- ... -->
     </dx-data-grid>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <!-- ... -->
+            <!-- The configuration repeats the previous code -->
+            <!-- ... -->
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.light.css';
+    import DxDataGrid, {
+        DxColumn,
+        DxLookup
+    } from 'devextreme-vue/data-grid';
+    import CustomStore from 'devextreme/data/custom_store';
+    import 'whatwg-fetch';
+
+    const isNotEmpty = (value) => value !== undefined && value !== null && value !== '';
+
+    const lookupDataSource = {
+        store: new CustomStore({
+            key: "id",
+            load: (loadOptions) => {
+                let params = '?';
+                [
+                    "sort", 
+                    // "skip",
+                    // "take", 
+                    // "filter", 
+                    // "searchExpr",
+                    // "searchOperation",
+                    // "searchValue",
+                    // "group"
+                ].forEach(function(i) {
+                    if(i in loadOptions && isNotEmpty(loadOptions[i])) {
+                        params += `${i}=${JSON.stringify(loadOptions[i])}&`;
+                    }
+                });
+                params = params.slice(0, -1);
+        
+                return fetch(`https://mydomain.com/MyDataService/authors${params}`)
+                    .then(response => response.json())
+                    .then(response => {
+                        return {
+                            data: response.data,
+                            totalCount: response.totalCount,
+                            summary: response.summary,
+                            groupCount: response.groupCount
```

Comment:
If this data source is used only for a lookup column, totalCount, summary and groupCount are redundant here. This is related to all lookup data sources in all approaches.